Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -1,30 +1,38 @@
 // Copyright 2014 Dolphin Emulator Project
 // SPDX-License-Identifier: GPL-2.0-or-later
 
+#include "InputCommon/GCAdapter.h"
+
 #include <algorithm>
 #include <array>
-#include <libusb.h>
 #include <mutex>
 #include <optional>
 
+#ifndef ANDROID
+#include <libusb.h>
+#else
+#include <jni.h>
+#endif
```

Comment:
I was mostly trying to put the non-android code on the top (though there isn't a strong reason for doing so).  I could add a second define for using the libusb implementation instead, or I could just swap them.  Do you have any preference between these?

Reply:
Almost called this out as well.  I agree about the positiveness being more readable though I also feel like `if` statements are easier to read if the smaller block is on top.

As for using a second define or swap them, I don't personally have a preference.

Reply:
In the long run I don't think it really matters because the plan is to split the android USB code and the libusb code into separate files again in a later PR (leaving the common logic in GCAdapter), so it doesn't really matter which comes first.  I'll swap them.

Reply:
... actually, while trying to swap them, I've found that there are a lot of cases where it's `#ifndef ANDROID` without a corresponding `#else` (at least initially), so a second define is probably better (otherwise it'll just end up being a mix of `#ifdef ANDROID` and `#ifndef ANDROID` which is much more confusing than almost entirely `#ifndef ANDROID`).